### PR TITLE
refactor: lowercasing tech stack name

### PR DESCRIPTION
**Pull request by s2sharpit**
## Related Issue

Closes: #1913 

## Description
Refactoring all the teck-stack name to lowercase

### CODE REVIEW

File: DB/projects.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
         "description": "getData of the github users",
         "tech": [
           "reactjs",
-          "Git"
+          "git"
```
Comment (priyankarpal) [inline DB/projects.json:42]: remove `git`
```suggestion
     
```

File: DB/projects.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
         "description": "getData of the github users",
         "tech": [
           "reactjs",
```
Comment (priyankarpal) [inline DB/projects.json:41]: remove the comma
```suggestion
          "reactjs"
```

File: DB/projects.json [hunk 3]
```diff
@@ -48,7 +48,7 @@
         "description": "FileDrop is an FileSharing Application ",
         "tech": [
           "reactjs",
-          "Nodejs",
+          "nodejs",
           "jwt",
           "js",
```
Comment (priyankarpal) [inline DB/projects.json:53]: change to `javascript`
```suggestion
          "javascript",
```

File: DB/projects.json [hunk 4]
```diff
@@ -140,7 +140,7 @@
           "javascript",
           "nodejs",
           "expressjs",
-          "mailchimpApi"
+          "mailchimpapi"
```
Comment (priyankarpal) [inline DB/projects.json:143]: change to `api`
```suggestion
          "api"
```

File: DB/projects.json [hunk 5]
```diff
@@ -338,7 +338,7 @@
           "javascript",
           "html",
           "css",
-          "Express",
+          "express",
```
Comment (priyankarpal) [inline DB/projects.json:341]:
```suggestion
          "expressjs",
```

File: DB/projects.json [hunk 6]
```diff
@@ -1335,8 +1335,8 @@
         "title": "JobSpot",
         "description": "A Mobile Application for both Android and iOS for Job Opportunities.",
         "tech": [
-          "reactjs Native",
-          "Expo"
+          "reactjs native",
```
Comment (priyankarpal) [inline DB/projects.json:1338]:
```suggestion
          "reactnative",
```

File: DB/projects.json [hunk 7]
```diff
@@ -1759,7 +1759,7 @@
         "description": "The project is a Google Chrome extension for adding Bookmarks of YouTube video timestamps. Users can bookmark a video at particular timestamp and resume watching it later.",
         "tech": [
           "javascript",
-          "DOM",
+          "dom",
```
Comment (priyankarpal) [inline DB/projects.json:1762]:
```suggestion

```

File: DB/projects.json [hunk 8]
```diff
@@ -1808,10 +1808,10 @@
...
         "tech": [
           "reactjs",
-          "MUI",
-          "reactjs Router",
-          "Context API",
-          "Google Custom Search API",
+          "mui",
+          "reactjs router",
+          "context api",
+          "google custom search api",
```
Comment (priyankarpal) [inline DB/projects.json:1814]:
```suggestion
          "mui",
          "api",
```

File: DB/projects.json [hunk 9]
```diff
@@ -1833,7 +1833,7 @@
           "javascript",
           "css",
           "html",
-          "ipify API"
+          "ipify api"
```
Comment (priyankarpal) [inline DB/projects.json:1836]:
```suggestion
          "api"
```

File: DB/projects.json [hunk 10]
```diff
@@ -1876,7 +1876,7 @@
           "javascript",
           "css",
           "html",
-          "Rapid API"
+          "rapid api"
```
Comment (priyankarpal) [inline DB/projects.json:1879]:
```suggestion
          "api"
```

File: DB/projects.json [hunk 11]
```diff
@@ -1897,7 +1897,7 @@
           "javascript",
           "css",
           "html",
-          "Weather-api",
+          "weather-api",
```
Comment (priyankarpal) [inline DB/projects.json:1900]:
```suggestion
          "api",
```

File: DB/projects.json [hunk 12]
```diff
@@ -2055,10 +2055,10 @@
         "description": "A fully working clone of YouTube made using reactjs, Material UI and YouTube v3 API. User can search and play the videos in the app directly.",
         "tech": [
           "reactjs",
-          "Material UI",
-          "reactjs Router",
+          "mui",
+          "reactjs router",
```
Comment (priyankarpal) [inline DB/projects.json:2059]:
```suggestion
          "reactjs",
```

File: DB/projects.json [hunk 13]
```diff
@@ -2055,10 +2055,10 @@
...
           "css",
-          "Youtube API"
+          "youtube api"
```
Comment (priyankarpal) [inline DB/projects.json:2061]:
```suggestion
          "api"
```

File: DB/projects.json [hunk 14]
```diff
@@ -2119,8 +2119,8 @@
         "description": "This a news API/Rss feed parser which scrapes news from websites and provides them in simple json format for the ease of developers. Read the docs and use the console page to live test the API.",
         "tech": [
           "reactjs",
-          "Material UI",
-          "Express js",
+          "mui",
+          "express js",
```
Comment (priyankarpal) [inline DB/projects.json:2123]:
```suggestion
          "expressjs",
```

File: DB/projects.json [hunk 15]
```diff
@@ -2162,8 +2162,8 @@
         "tech": [
           "javascript",
           "reactjs",
-          "Node",
-          "RapidApi",
+          "node",
+          "rapidapi",
```
Comment (priyankarpal) [inline DB/projects.json:2166]:
```suggestion
          "api",
```

File: DB/projects.json [hunk 16]
```diff
@@ -2287,8 +2287,8 @@
         "tech": [
           "mongodb",
           "reactjs",
-          "Node",
-          "Express"
+          "node",
+          "express"
```
Comment (priyankarpal) [inline DB/projects.json:2291]:
```suggestion
          "nodejs",
          "expressjs"
```

File: DB/projects.json [hunk 17]
```diff
@@ -2377,11 +2377,11 @@
         "description": "A bot that generates text and images for the given prompts.",
         "tech": [
           "reactjs",
-          "Node",
-          "Express",
+          "node",
+          "express",
```
Comment (priyankarpal) [inline DB/projects.json:2381]:
```suggestion
          "nodejs",
          "expressjs",
```

File: DB/projects.json [hunk 18]
```diff
@@ -2377,11 +2377,11 @@
...
           "javascript",
           "css",
-          "OpenAi API"
+          "openai api"
```
Comment (priyankarpal) [inline DB/projects.json:2384]:
```suggestion
          "api"
```

File: DB/projects.json [hunk 19]
```diff
@@ -2403,7 +2403,7 @@
         "tech": [
           "reactjs js",
           "tailwindcss",
-          "Rapid API",
+          "rapid api",
```
Comment (priyankarpal) [inline DB/projects.json:2406]:
```suggestion
          "api",
```

File: DB/projects.json [hunk 20]
```diff
@@ -2446,8 +2446,8 @@
           "html",
           "css",
           "javascript",
-          "Fetch API",
-          "JSON"
+          "fetch api",
+          "json"
```
Comment (priyankarpal) [inline DB/projects.json:2450]:
```suggestion
          "api"
```

File: DB/projects.json [hunk 21]
```diff
@@ -2686,8 +2686,8 @@
         "tech": [
           "reactjs",
           "tailwindcss",
-          "OpenAI API",
-          "Typescript"
+          "openai api",
```
Comment (priyankarpal) [inline DB/projects.json:2689]:
```suggestion
          "api",
```

File: DB/projects.json [hunk 22]
```diff
@@ -2706,13 +2706,13 @@
         "title": "AR Watches",
         "description": "The AR Watches Android app is designed to showcase augmented reality technology by allowing users to virtually try on watches using their smartphone's camera.",
         "tech": [
-          "Android Studio",
+          "android studio",
```
Comment (priyankarpal) [inline DB/projects.json:2709]:
```suggestion

```

File: DB/projects.json [hunk 23]
```diff
@@ -2770,8 +2770,8 @@
         "title": "DePay",
         "description": "A web3 based decentralised money lending system to allow users to send,receive,borrow and lend money(on Goerli Testnet)",
         "tech": [
-          "NextJs",
-          "Solidity",
+          "nextJs",
```
Comment (priyankarpal) [inline DB/projects.json:2773]:
```suggestion
          "nextjs",
```

File: DB/projects.json [hunk 24]
```diff
@@ -3376,7 +3376,7 @@
         "description": "Wave is a Discord bot which is made using Python to resolve the problem which is an external software requirement for recording voice chats of DIscord channels, this discord bot aims enabling you to record the conversations and can provide you the recording of chat",
         "tech": [
           "python",
-          "Discord API"
+          "discord api"
```
Comment (priyankarpal) [inline DB/projects.json:3379]:
```suggestion
          "api"
```

File: DB/projects.json [hunk 25]
```diff
@@ -3768,9 +3768,9 @@
         "description": "A Discord Bot for Coders. Now Contests and Hackathons related Information under your command. Executes Code in Discord Itself. Provides Jokes, News, Quotes and fun things. Supported - C, c++, Python, java, javascript, Golang, C# etc.",
         "tech": [
           "nodejs",
-          "Express.js",
-          "Discord.js",
-          "RapidApi"
+          "expressjs",
+          "discordjs",
+          "rapidapi"
```
Comment (priyankarpal) [inline DB/projects.json:3773]:
```suggestion
          "api"
```

File: DB/projects.json [hunk 26]
```diff
@@ -3960,7 +3960,7 @@
           "javascript",
           "css",
           "html",
-          "RapidAPI",
+          "rapidapi",
```
Comment (priyankarpal) [inline DB/projects.json:3963]:
```suggestion
          "api",
```

File: DB/projects.json [hunk 27]
```diff
@@ -4216,10 +4216,10 @@
         "description": "This is a replica pf other movies platform where you can get details and related details of the Movies and Tv shows",
         "tech": [
           "html",
-          "TAILWIND",
-          "REACT",
-          "RAPIDAPI",
-          "JAVASCRIPT"
+          "tailwindcss",
+          "reactjs",
+          "rapid api",
```
Comment (priyankarpal) [inline DB/projects.json:4221]:
```suggestion
          "api",
```

File: DB/projects.json [hunk 28]
```diff
@@ -4254,10 +4254,10 @@
         "tech": [
           "html",
           "css",
-          "reactjs js",
-          "Styled component",
-          "Context Api",
-          "Email js Api"
+          "reactjs",
+          "styled component",
+          "context api",
```
Comment (priyankarpal) [inline DB/projects.json:4259]:
```suggestion
```

File: DB/projects.json [hunk 29]
```diff
@@ -4254,10 +4254,10 @@
...
+          "emailjs api"
```
Comment (priyankarpal) [inline DB/projects.json:4260]:
```suggestion
          "api"
```

File: DB/projects.json [hunk 30]
```diff
@@ -4319,7 +4319,7 @@
         "title": "LetsNote",
         "description": "This is a note-taking web application that is reactjs Based",
         "tech": [
-          "Express",
+          "express",
```
Comment (priyankarpal) [inline DB/projects.json:4322]:
```suggestion
          "expressjs",
```

File: DB/projects.json [hunk 31]
```diff
@@ -4407,10 +4407,10 @@
         "description": "This is a replica pf other movies platform where you can get details and related details of the Movies and Tv shows",
         "tech": [
           "html",
-          "TAILWIND",
-          "REACT",
-          "RAPIDAPI",
-          "JAVASCRIPT"
+          "tailwind",
+          "react",
+          "rapidapi",
```
Comment (priyankarpal) [inline DB/projects.json:4412]:
```suggestion
          "tailwindcss",
          "reactjs"
```

File: DB/projects.json [hunk 32]
```diff
@@ -4521,9 +4521,9 @@
         "description": "This is simply a news app which keeps you updated with the latest news and not only the latest but the most relevant one ",
         "tech": [
           "html",
-          "TAILWIND",
-          "REACT",
-          "JAVASCRIPT"
+          "tailwind",
+          "react",
```
Comment (priyankarpal) [inline DB/projects.json:4525]:
```suggestion
          "tailwindcss",
          "reactjs",
```

File: DB/projects.json [hunk 33]
```diff
@@ -4544,11 +4544,11 @@
...
         "tech": [
           "html",
-          "MATERIALUI",
-          "REACT",
+          "mui",
+          "reactjs",
           "css",
-          "COINGECKOAPI",
-          "JAVASCRIPT"
+          "coingeckoapi",
```
Comment (priyankarpal) [inline DB/projects.json:4550]:
```suggestion
          "api",
```

File: DB/projects.json [hunk 34]
```diff
@@ -4758,11 +4758,11 @@
...
-          "BootStrap",
-          "reactjs JS",
-          "useContext",
-          "API",
-          "Vercel"
+          "bootstrap",
+          "reactjs",
+          "usecontext",
+          "api",
+          "vercel"
```
Comment (priyankarpal) [inline DB/projects.json:4765]:
```suggestion
          "api"
```

File: DB/projects.json [hunk 35]
```diff
@@ -4917,8 +4917,8 @@
           "reactjs",
           "html",
           "css",
-          "Material UI",
-          "IMDB API"
+          "mui",
+          "imdb api"
```
Comment (priyankarpal) [inline DB/projects.json:4921]:
```suggestion
          "api"
```

File: DB/projects.json [hunk 36]
```diff
@@ -5115,8 +5115,8 @@
         "title": "CocktailDB",
         "description": "CocktailDB is a web application that allows users to explore a vast collection of cocktails, along with their prices, ingredients, and details",
         "tech": [
-          "reactjs JS",
-          "Rapid API",
+          "reactjs",
+          "rapid api",
```
Comment (priyankarpal) [inline DB/projects.json:5119]:
```suggestion
          "api",
```

File: DB/projects.json [hunk 37]
```diff
@@ -5125,8 +5125,8 @@
         "title": "job-update-reactNative-app",
         "description": "JobPilot is a cross-platform job update app built using reactjs-Native that allows you to Ultra-Fast and Simple Job Search for jobs posted on LinkedIn, Indeed, Glassdoor, ZipRecruiter, BeBee and many others, all in a single API. Check working screenshot below",
         "tech": [
-          "reactjs Native",
-          "Expo",
+          "reactjs native",
```
Comment (priyankarpal) [inline DB/projects.json:5128]:
```suggestion
          "reactnative",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: DB/projects.json [hunk 38]
```diff
@@ -1537,8 +1537,8 @@
           "css",
           "javascript",
           "html",
-          "TMD API",
-          "Youtube API"
+          "api",
+          "api"
```
Comment (priyankarpal) [inline DB/projects.json:1541]: remove one `api`
```suggestion

```

File: DB/projects.json [hunk 39]
```diff
@@ -1854,8 +1850,8 @@
         "tech": [
           "reactjs",
           "javascript",
-          "TailWindCss",
-          "Sanity "
+          "tailWindCss",
```
Comment (priyankarpal) [inline DB/projects.json:1853]:
```suggestion
          "tailwindcss",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: DB/projects.json [hunk 40]
```diff
@@ -4407,10 +4399,9 @@
         "description": "This is a replica pf other movies platform where you can get details and related details of the Movies and Tv shows",
         "tech": [
           "html",
-          "TAILWIND",
-          "REACT",
-          "RAPIDAPI",
-          "JAVASCRIPT"
+          "tailwindcss",
+          "reactjs"
```
Comment (priyankarpal) [inline DB/projects.json:4403]: add a comma
```suggestion
          "reactjs",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]

**Comment by s2sharpit** [comment]
There are some issues still. Let me fix them.

**Pull request merged**